internal/controllers/clusterextension_controller_test.go

```diff
@@ -186,8 +190,17 @@ func TestClusterExtensionBundleDeploymentDoesNotExist(t *testing.T) {
 }
 
 func TestClusterExtensionChannelVersionExists(t *testing.T) {
-	t.Skip("Skip and replace with e2e for BundleDeployment")
 	cl, reconciler := newClientAndReconciler(t)
+	mockUnpacker := unp.(*MockUnpacker)
```

Comment:
See comment re: `unp` declaration, as this seems like it might be a local variable, but it's not.

Reply:
refactored now